QuickWins.rst

```diff
@@ -17,3 +17,21 @@ Starting notes:
 
 Hack-a-thon content:
 --------------------
+
+* Play politics (a little bit) 
+
+It can't hurt to start to understand the politics or culture of the organization. Is this an Engineering-centric company, where Marketing is an afterthought? Or does the organization lead with Marketing? Is the Customer Success/Support function adequately staffed? Reaching out to principals in Marketing, Engineering, Support, Operations, etc. will help you eastablish and reinforce your role. When interviewing these principals, you may hear sentiments ranging from, "Boy am I glad you're here!" to "Who are you again (and why are you wasting my time)?" Focusing your efforts, at least at first, on those functions in the organization who appeciate the motivation for filling your role will guide you to a sympathic audience (who, in turn, can become internal allies)! You may have a broad charter or a narrow one; in either case, a quick win should be focused to those teams who appreciate your presence. 
+
+• Triage
+
+Seek out and embrace the triage process; your quick wins ought to be focused on "Severity 1 / Priority 1" issues. "Stopping the bleeding" is a quick win itself! Insist on attending bug triages (or backlog review, or any sort of formal prioritization process or bug scrub); doing so will immerse you quickly into the pain points of the product(s) and help you envision and prioritize first deliverables. (Note that the highest source of pain may be customer-facing or internal. You may not necessarily have a charter to produce internal documentation, but if you are alleviating pain, you could make a case for producing something quickly for an internal audience. See above.)
```

Comment:
Suggestion:

"doing so will immerse you quickly into" -> "doing so will quickly immerse you into"

Reply:
sure